src/Language/Haskell/Brittany/Internal/Layouters/IE.hs

```diff
@@ -122,24 +119,20 @@ layoutLLIEs enableSingleline llies = do
   ieDs        <- layoutAnnAndSepLLIEs llies
   hasComments <- hasAnyCommentsBelow llies
   case ieDs of
-    [] -> docAltFilter
-      [ (not hasComments, docLit $ Text.pack "()")
-      , ( hasComments
-        , docPar (docSeq [docParenLSep, docWrapNodeRest llies docEmpty])
-                 docParenR
-        )
-      ]
-    (ieDsH:ieDsT) -> docAltFilter
-      [ ( not hasComments && enableSingleline
-        , docSeq
+    [] -> runFilteredAlternative $ do
```

Comment:
Can factor out `runFilteredAlternative $` out of the `case`s.

Reply:
Done.